### ISSUE TITLE
New version of django-otp breaks django-two-factor-auth

### ISSUE THREAD

**Issue by moggers87** [Bouke/django-two-factor-auth#122]
[See django-otp changlog](https://pythonhosted.org/django-otp/changes.html#v0-3-2-october-11-2015-django-1-8)

Example traceback:
```
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/home/moggers/.virtualenvs/inboxen/lib/python2.7/site-packages/django/core/management/__init__.py", line 351, in execute_from_command_line
    utility.execute()
  File "/home/moggers/.virtualenvs/inboxen/lib/python2.7/site-packages/django/core/management/__init__.py", line 325, in execute
    django.setup()
  File "/home/moggers/.virtualenvs/inboxen/lib/python2.7/site-packages/django/__init__.py", line 18, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/moggers/.virtualenvs/inboxen/lib/python2.7/site-packages/django/apps/registry.py", line 108, in populate
    app_config.import_models(all_models)
  File "/home/moggers/.virtualenvs/inboxen/lib/python2.7/site-packages/django/apps/config.py", line 198, in import_models
    self.models_module = import_module(models_module_name)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/home/moggers/.virtualenvs/inboxen/lib/python2.7/site-packages/two_factor/models.py", line 9, in <module>
    from django_otp import Device
ImportError: cannot import name Device
```

**Comment by moggers87** [comment]
OK, this appears to be fixed in master. Just needs a new release to solve this issue.

**Comment by Bouke** [comment]
This has already been resolved in master (04a09fa60134b72687a129a3cd1f3e553cd6ed7d). I've just published a patch release (1.2.1) as an intermediate fix.

**Comment by moggers87** [comment]
Thanks! ☺
